modules/openapi-generator/src/main/resources/python/configuration.mustache

```diff
@@ -60,17 +60,22 @@ class Configuration(object):
         """Password for HTTP basic authentication
         """
 {{#hasOAuthMethods}}
-        self.access_token = ""
+        self.access_token = None
```

Comment:
Down below this is added to a string when we return r=from self.auth_settings()
can you keep this as a string default so it is impossible to throw an exception?
How about passing `access_token = ''` as an init arg, and setting
`self.access_token = access_token`?
That way we have that line only once in the {{#hasBearerMethods}} tag.

Reply:
My reasoning for having both None and empty string was a concern some existing SDK users may complain this would break compatibility. For example, if somebody is using the SDK to test a system, they would probably want to test with empty user, empty password, empty access token, so I thought it wouldn't be a good idea to omit the authorization attribute (e.g. query, cookie, header) just because a string is empty.